UnityProjects/MRTKDevTemplate/Assets/Prefabs/SeeItSayIt/AdjustLabelPosition.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos
+{
+
+    /// <summary>
+    /// Script to adjust the position of a another GameObject when this GameObject resizes.
+    /// EX Used to position see-it say-it labels when non-canvas button backplate resizes. 
+    /// </summary>
+    public class AdjustLabelPosition : MonoBehaviour
+    {
+
+        [SerializeField]
+        private Transform label;
```

Comment:
Add a tooltip for this serialized field, with a brief description about the label

Reply:
An serialized field or public/protected member needs to be documented.